src/content_script.js

```diff
@@ -1,5 +1,22 @@
 "use strict";
 
+function detectFacebookLoginButton() {
+  // Test for standard implementation (Example: Facebook Docs)
+  const loginButton = document.querySelectorAll(".fb-login-button");
+  for (let item of loginButton) {
+    const fbcClassResetArr = ["fbc-overlay-small", "fbc-overlay"];
+    let replacementClassArr = ["fbc-loginButton"];
```

Comment:
prefer `const`

Reply:
Revised. I left the `for (let item…` stand as is. However, it looks like [it should work](https://stackoverflow.com/questions/31987465/ecmascript-2015-const-in-for-loops) as a `const`. I can replace both this line and the two loops in the `detectFacebookOnPage()`  function.